src/test/java/io/specto/hoverfly/ruletest/CaptureOrSimulationModeTest.java

```diff
@@ -0,0 +1,28 @@
+package io.specto.hoverfly.ruletest;
+
+import io.specto.hoverfly.junit.core.HoverflyMode;
+import io.specto.hoverfly.junit.rule.HoverflyRule;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CaptureOrSimulationModeTest {
+
+    @Test
+    public void shouldInstantiateHoverflyInCaptureModeInCaseOfNoRecord() {
+
+        final HoverflyRule hoverflyRule = HoverflyRule.inCaptureOrSimulationMode("mynewservicerecord.json");
```

Comment:
But is this needed? I mean the only thing that I am testing is that instances are the correct ones, I am not executing anything. What I am validating in this test is that if file exists then one rune configured with simulate mode is returned and if not one with capture. I don't care in this test how both modes works since it is tested in another test.